src/lib/engine.js

```diff
@@ -378,6 +382,51 @@ assign(Engine.prototype, EventEmitter.prototype, {
     return r;
   },
 
+  updatePicture(file) {
+    return new Promise((resolve, reject) => {
+      let s3config = Hull.config().services.storage.hull;
+
+      let url = s3config.url;
+      let type = file.type;
+      let name = file.name;
+
+      let form = new FormData();
```

Comment:
FormData is only supported on IE10+ 

http://caniuse.com/#feat=xhr2
https://developer.mozilla.org/en-US/docs/Web/API/FormData